src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.cs

```diff
@@ -248,27 +249,20 @@ private static int LocateFirstFoundByte(ulong match)
                                                0x02ul << 40 |
                                                0x01ul << 48) + 1;
 
-        public static bool TryGetEscapedGuid(ReadOnlySpan<byte> span, out Guid value)
+        public static void GetValueSequenceAsSpan(ReadOnlySequence<byte> valueSequence, out ReadOnlySpan<byte> span)
```

Comment:
What is the intended behavior of this method? The current behavior is that it copies some data within the local frame's buffer, outs a pointer to the local frame, then allows the local frame to unwind. Once the method returns `span` will always point to uninitialized memory.

Reply:
The intention was to copy the contents of the reader's `ValueSequence` to a `span`, then `out` it. Since we can't `out` a `span` that has been stackalloc'd, I've opted to inline this logic in the previously calling methods.